spec.md

```diff
@@ -1641,7 +1641,7 @@ wire pred: UInt<1>
 wire en: UInt<1>
 pred <= eq(X, Y)
 en <= Z_valid
-cover(clk, pred, en, "X equals Y when Z is valid") : optional_name
+cover(clk, pred, en) : optional_name
```

Comment:
are these completely missing from the grammar at the end? should this PR add this (along with assert and assume) to formalize this?

Reply:
yes, I'm working on clear them up.

Reply:
I just found BNF doesn't include verification statements. I'll create another PR for it.